python/cudf/cudf/core/frame.py

```diff
@@ -3812,3 +3813,37 @@ def _is_series(obj):
     instead of checking for isinstance(obj, cudf.Series)
     """
     return isinstance(obj, Frame) and obj.ndim == 1 and obj._index is not None
+
+
+def _drop_rows_by_labels(
+    obj: Union[cudf.DataFrame, cudf.Series],
+    labels: Union[ColumnLike, abc.Iterable, str],
+    level: Union[int, str],
+    errors: str,
+) -> Union[cudf.DataFrame, cudf.Series]:
+    """Remove rows specified by `labels`. If `errors=True`, an error is raised
+    if some items in `labels` do not exist in `obj._index`.
+
+    Parameter `level` is ignored if `obj._index` is not `MultiIndex`
```

Comment:
Rather than ignoring `level`, should we just raise when `level >= obj._index.nlevels`? For a flat index, `.nlevels = 0`.